var/spack/repos/builtin/packages/wrf/package.py

```diff
@@ -15,132 +60,309 @@ class Wrf(Package):
     for both atmospheric research and operational forecasting applications.
     """
 
-    homepage = "https://www.mmm.ucar.edu/weather-research-and-forecasting-model"
-    url      = "https://github.com/wrf-model/WRF/archive/v4.2.tar.gz"
-    maintainers = ['MichaelLaufer']
+    homepage = (
+        "https://www.mmm.ucar.edu/weather-research-and-forecasting-model"
+    )
+    url = "https://github.com/wrf-model/WRF/archive/v4.2.tar.gz"
+    maintainers = ["ptooley"]
```

Comment:
Can you add your name to the list of maintainers instead of replacing @MichaelLaufer?

Reply:
Yes, sorry. This started life as a new recipe because the changes were so big.